### DISCUSSION TITLE
Rare symbol/▽ should distinguish between forms and readings

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#113] [Errors and Corrections]
This is just a nitpick and not related to the new symbol changes, but shouldn't the table distinguish between rare forms and readings (or both)?
In the table for 氷室, if you hover on ひょうしつ it says "rarely used form" when "rarely used reading" would be more correct.
I guess it could get annoying to check if there are both [rK] and [rk] tags to display something like "rarely used form and reading" though.

**Comment by parfait8566** [comment]
![image](https://github.com/user-attachments/assets/fef19d83-d71d-4ddc-8976-6d3208b7ae2a)
Also, in cases like this it might be more intuitive to just say "valid form".

**Comment by stephenmk** [comment]
Yes, I should take another look at these tooltips.
